common-develocity-gradle-configuration-groovy/settings.gradle

```diff
@@ -1,18 +1,16 @@
 plugins {
-    id 'com.gradle.enterprise' version '3.17'
+    id 'com.gradle.develocity' version '3.17'
     id 'com.gradle.common-custom-user-data-gradle-plugin' version '2'
 }
 
 def isCI = System.getenv('CI') != null // adjust to your CI provider
 
-gradleEnterprise {
+develocity {
     server = 'https://develocity-samples.gradle.com' // adjust to your Develocity server
     allowUntrustedServer = false // ensure a trusted certificate is configured
 
     buildScan {
```

Comment:
I suspect there is no hope in convincing you all of replacing:

```groovy
buildScan {
    uploadInBackground = !isCI
}
```

With:

```groovy
buildScan.uploadInBackground = !isCI

```

🙃